api/src/pcapi/sandboxes/scripts/creators/industrial/create_offerer_providers_for_apis.py

```diff
@@ -39,25 +45,50 @@ def create_offerer_provider(
 
 def create_offerer_provider_with_offers(name: str) -> None:
     in_five_days = datetime.datetime.utcnow().replace(second=0, microsecond=0) + datetime.timedelta(days=5)
-    offerer, provider = create_offerer_provider(name)
-    first_venue = offerers_facotries.VenueFactory(name="Zénith de Lisieux", managingOfferer=offerer)
-    second_venue = offerers_facotries.VenueFactory(name="Olympia de Besançon", managingOfferer=offerer)
+    offerer, provider = create_offerer_provider(name, provider_name="TaylorManager")
+    user = users_factories.ProFactory(
+        email="taylor@example.com",
+        firstName="taylor",
+        lastName="young",
+        phoneNumber="+33100000000",
+        password=crypto.hash_password("user@AZERTY123"),
+    )
+    offerers_factories.UserOffererFactory(offerer=user, user=user)
+    first_venue = offerers_factories.VenueFactory(name="Zénith de Lisieux", managingOfferer=offerer)
+    second_venue = offerers_factories.VenueFactory(name="Olympia de Besançon", managingOfferer=offerer)
     providers_factories.VenueProviderFactory(venue=first_venue, provider=provider)
     providers_factories.VenueProviderFactory(venue=second_venue, provider=provider)
-    first_offer = offers_factories.EventOfferFactory(
-        name="Taylor à Besançon !",
-        venue=first_venue,
-        subcategoryId=subcategories.CONCERT.id,
-        lastProvider=provider,
-    )
-    offers_factories.EventStockFactory(
-        offer=first_offer,
-        beginningDatetime=in_five_days,
-    )
-    offers_factories.EventStockFactory(
-        offer=first_offer,
-        beginningDatetime=in_five_days + datetime.timedelta(days=1),
-    )
+    offers = []
+    for i in range(10):
+        offer = offers_factories.EventOfferFactory(
+            name=f"Taylor à {i}",
```

Comment:
```suggestion
            name=f"Taylor à Lisieux {i}",
```
est-il plutôt ce que tu voulais, et non « Taylor à 1 » ?

Reply:
Effectivement